InstaBotPrototype/DBMigrator/DBMigration.cs

```diff
@@ -14,13 +14,27 @@ public DBMigration(DbProviderFactory factory, DbConnection connection)
             ReverseCommand = Factory.CreateCommand();
 
             this.connection = connection;
+
+            var command = factory.CreateCommand();
+            command.CommandText = $"select count(*) from dbo.Migrations where Name = '{Name}'";
+            command.Connection = connection;
+
+            try
+            {
+                if ((int)command.ExecuteScalar() == 1)
+                {
+                    IsApplied = true;
+                }
+
+            }
+            catch { }
```

Comment:
its very disappointing to see empty catch. its better not to handle exception at all so we can catch it somewhere upper. or add some output or log so user can see he as some problem with db. cause its bad approach just to hide exceptions